### ISSUE TITLE
`[]` around the first line of every executed command

### ISSUE THREAD

**Issue by emiljohansson** [sindresorhus/pure#211]
I recently installed OSX El Capitan and since the update every executed command is getting wrapped in in brackets `[]`. Only the first line is getting the brackets and only after executing the command.
My colleague recently installed Pure for the first time and are seeing the same thing.

<img width="383" alt="screen shot 2016-05-23 at 9 50 24 am" src="https://cloud.githubusercontent.com/assets/1788399/15476441/21acb66a-20cc-11e6-8eb7-6a7dd05e6815.png">

Thanks,
Emil

**Comment by sindresorhus** [comment]
Duplicate of https://github.com/sindresorhus/pure/issues/203.
